dataset.cfg

```diff
@@ -0,0 +1,3 @@
+[DATASET]
```

Comment:
should delete this file, and write `dataset.cfg` into `.gitignore`

Reply:
added